features/call/impl/src/main/kotlin/io/element/android/features/call/impl/pip/PipSupportProvider.kt

```diff
@@ -34,9 +37,15 @@ interface PipSupportProvider {
 @ContributesBinding(AppScope::class)
 class DefaultPipSupportProvider @Inject constructor(
     @ApplicationContext private val context: Context,
+    private val featureFlagService: FeatureFlagService,
 ) : PipSupportProvider {
     override fun isPipSupported(): Boolean {
-        val hasSystemFeaturePip = context.packageManager?.hasSystemFeature(PackageManager.FEATURE_PICTURE_IN_PICTURE).orFalse()
-        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && hasSystemFeaturePip
+        val isSupportedByTheOs = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O &&
+            context.packageManager?.hasSystemFeature(PackageManager.FEATURE_PICTURE_IN_PICTURE).orFalse()
+        return if (isSupportedByTheOs) {
+            runBlocking { featureFlagService.isFeatureEnabled(FeatureFlags.PictureInPicture) }
```

Comment:
I think this is OK to use a `runBlocking` here, since at some point the FF will be removed.